app/src/main/java/com/example/habitsmasher/DisplaysErrorMessages.java

```diff
@@ -0,0 +1,16 @@
+package com.example.habitsmasher;
+
+/**
+ * Interface implemented by any UI class that displays error messages
+ * @author Jason Kim
+ */
+public interface DisplaysErrorMessages {
+
+    /**
+     * Displays the requested error message
+     * @param messageType code indicating the type of error message to
+     *                    be displayed
+     */
+    void displayErrorMessage(int messageType);
+}
+
```

Comment:
This interface is implemented by any dialog/UI class that displays error messages. It is used in the validators of Habit/Habit Events.